doc/API.md

```diff
@@ -711,253 +711,190 @@ The input byte array should be 80 bytes that form the solved block header. *Unli
+| ------------------------------------------------------------- | --------- |
+| [/renter](#renter-get)                                        | GET       |
+| [/renter](#renter-post)                                       | POST      |
+| [/renter/contracts](#rentercontracts-get)                     | GET       |
+| [/renter/downloads](#renterdownloads-get)                     | GET       |
+| [/renter/files](#renterfiles-get)                             | GET       |
+| [/renter/delete/___*siapath___](#renterdeletesiapath-post)    | POST      |
+| [/renter/download/___*siapath___](#renterdownloadsiapath-get) | GET       |
+| [/renter/rename/___*siapath___](#renterrenamesiapath-post)    | POST      |
+| [/renter/upload/___*siapath___](#renteruploadsiapath-post)    | POST      |
 
-#### /renter/allowance [GET]
+For examples and detailed descriptions of request and response parameters,
+refer to [Renter.md](/doc/api/Renter.md).
 
-Function: Returns the current contract allowance.
+#### /renter [GET]
 
-Parameters: none
+returns the current settings along with metrics on the renter's spending.
 
-Response:
-```
-struct {
-	funds  types.Currency    (string)
-	hosts  uint64
-	period types.BlockHeight (uint64)
+###### JSON Response [(with comments)](/doc/api/Renter.md#json-response)
+```javascript
+{
+  "settings": {
+    "allowance": {
+      "funds":       "1234", // hastings
+      "hosts":       24,
+      "period":      6048, // blocks
+      "renewwindow": 3024  // blocks
+    }
+  },
+  "financialmetrics": {
+    "contractspending": "1234", // hastings
+    "downloadspending": "5678", // hastings
+    "storagespending":  "1234", // hastings
+    "uploadspending":   "5678"  // hastings
+  }
 }
 ```
-'funds' is the number of hastings allocated for file contracts in the given
-period.
-
-'hosts' is the number of hosts that contracts will be formed with.
 
-'period' is the duration of contracts formed.
+#### /renter [POST]
 
-#### /renter/allowance [POST]
+modify settings that control the renter's behavior.
 
-Function: Sets the contract allowance.
-
-Parameters: none
+###### Query String Parameters [(with comments)](/doc/api/Renter.md#query-string-parameters)
 ```
-funds  types.Currency    (string)
-hosts  uint64
-period types.BlockHeight (uint64)
+funds  // (optional) hastings
+hosts  // (optional)
+period // (optional) block height
 ```
-'funds' is the number of hastings allocated for file contracts in the given
-period.
-
-'hosts' is the number of hosts that contracts will be formed with.
 
-'period' is the duration of contracts formed.
-
-Response: standard
-
-#### /renter/downloads [GET]
+###### Response
+standard success or error response. See
+[API.md#standard-responses](/doc/API.md#standard-responses).
 
-Function: Lists all files in the download queue.
+#### /renter/contracts [GET]
 
-Parameters: none
+// TODO: is this description correct?
+returns all contracts that have been formed by the renter.
```

Comment:
Does this route return all contracts that have ever been formed by the renter or only currently used & open contracts?